source/text_handler.h

```diff
@@ -263,10 +263,9 @@ class AssemblyContext {
     // It also allows octal input, but we don't care about that case.
     text_stream >> std::setbase(0);
     text_stream >> *value_pointer;
-    bool ok = true;
 
     // We should have read something.
-    ok = (text[0] != 0) && !text_stream.bad();
+    bool ok = (text[0] != 0) && !text_stream.bad();
```

Comment:
Not weird at all -- in fact, the style guide [requires](https://google.github.io/styleguide/cppguide.html#Local_Variables) it.